config/releases.exs

```diff
@@ -70,13 +62,11 @@ config :elixir_boilerplate, ElixirBoilerplateWeb.Endpoint,
 
 config :elixir_boilerplate, Corsica, origins: Environment.get_list_or_first_value("CORS_ALLOWED_ORIGINS")
 
-if Environment.exists?("BASIC_AUTH_USERNAME") do
```

Comment:
Is this a _necessary_ change? I’m not against it but I’d like to know why we’re doing it 😄

Reply:
It was the only place we were using `exists?/1`. 

I thought the double conditional as we also have an `if` block in [`endpoint.ex`](https://github.com/mirego/elixir-boilerplate/pull/67/files#diff-cf7a255e84ed6a6f361bacb92237f7c4L93-R94) was useless.